cmd/d2/main.go

```diff
@@ -187,8 +189,13 @@ func compile(ctx context.Context, ms *xmain.State, plugin d2plugin.Plugin, theme
 		return nil, err
 	}
 
+	layout := plugin.Layout
+	// TODO: remove, this is just a feature flag to test sequence diagrams as we work on them
+	if os.Getenv("D2_SEQUENCE") == "1" {
```

Comment:
interesting. maybe we should have an EXPERIMENTS flag. I could see users wanting to use this even if the support is early and unstable. I'll make an issue on this.

Reply:
https://github.com/terrastruct/d2/issues/241